drivers/dma/dma_stm32.c

```diff
@@ -594,8 +595,11 @@ static int dma_stm32_init(const struct device *dev)
 	const struct dma_stm32_config *config = dev->config;
 	const struct device *clk = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
 
+	/* This isn't called because dmamux is the master */
+	LOG_ERR("dma_stm32_init");
```

Comment:
This also needs to be removed with the rest of the changes to this file.

Reply:
restored original file